examples/src/layer-tests/src/lib.pax

```diff
@@ -1,21 +1,28 @@
 
-<Rectangle width=50%
-     fill={hsl(ticks + 100, 50%, 50%)}
-    corner_radii={RectangleCornerRadii::radii(10.0,10.0,10.0,10.0)}
-/>
-if self.activated {
-     <Text text="This is some long text" class=centered id=text />
-}
-<Rectangle x=50% width=50%  @click=self.toggle
-     fill={hsl(ticks, 50%, 50%)}
-    corner_radii={RectangleCornerRadii::radii(10.0,10.0,10.0,10.0)}
-/>
+<Frame anchor_x=50% x=50% anchor_y=50% y=50% width=50% height=50%>
```

Comment:
When running this example I noticed that text selection wasn't working? Can we re-enable it?

Reply:
We disabled this because of the designer (since moving the mouse while down was causing random selections in the view). But I can re-enable it!

Reply:
let's re-enable it so people will be able to select text at launch and figure out a more surgical solution for the designer.